### DISCUSSION TITLE
Switch events logs from DEBUG to INFO

### DISCUSSION THREAD

**Discussion by drustan** [keycloak/keycloak#19433] [Q&A]
Hello!

I've spent too much time trying to wrap my head around how to turn events logs into INFO level.

I'm running Keycloak 20.0.5 and my log related configuration in conf/keycloak.conf is the following :

```
log = console
log-level = INFO,org.keycloak.events:DEBUG
log-console-output = json
```

By default, events are shown in DEBUG :

`kubectl logs -f -l=app.kubernetes.io/component=keycloak
[…]
{"timestamp":"2023-03-30T11:17:47.276+02:00","sequence":8441,"loggerClassName":"org.jboss.logging.Logger","loggerName":"org.keycloak.events","level":"DEBUG","message":"type=LOGIN, realmId=dfc01763-0r7b-49a3-fsde-20e8350x4176, clientId=netbox, userId=d8f3-sg73-471c-7g5d-dc9184b4be2e, ipAddress=192.168.1.1, auth_method=openid-connect, auth_type=code, response_type=code, redirect_uri=https://netbox.example.net/oauth/complete/keycloak/, consent=no_consent_required, code_id=8f6e5c6g-48a5-4db4-a45d-2h4j37462537, username=foo, response_mode=query, authSessionParentId=7g36-48a5-4db4-a45d-4e5512993087, authSessionTabId=hDkjfhgdJE","threadName":"executor-thread-0","threadId":15,"mdc":{},"ndc":"","hostName":"keycloak-2","processName":"QuarkusEntryPoint","processId":1}
[…]`

To switch that to INFO, I added a conf/quarkus.properties file with the following content :

`quarkus.log.category."org.keycloak.events".level=INFO`

Unfortunately, after that, the events are not shown either in DEBUG or INFO.

Am I missing something here?

Thanks for your help.

**Comment by sschu** [comment]
We added the following to our keycloak.conf:
```
#events logging, configures on which level success and error events are emitted
spi-events-listener-jboss-logging-enabled=true
spi-events-listener-jboss-logging-success-level=INFO
spi-events-listener-jboss-logging-error-level=WARN
```
